users/arkag/arkag.h

```diff
@@ -0,0 +1,159 @@
+#pragma once
+
+#include "quantum.h"
+#include "process_keycode/process_tap_dance.h"
+
+#define _QWERTY  0
```

Comment:
You may want to use an enum here instead, as it may may make things easier in the future.